content/pt/_index.html

```diff
@@ -47,7 +47,7 @@ <h2>Os desafios da migração de mais de 150 micro serviços para o Kubernetes</
         <br>
         <br>
         <br>
-        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">KubeCon em Shanghai em June 24-26, 2019</a>
+        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2019" button id="desktopKCButton">KubeCon em Shanghai em Junho, 24-26 de 2019</a>
```

Comment:
Hi @rodrigoporcionato .
Would you check this link again (See the `en` version)?
I think these links are updated as the new KubeCon dates are announced.